### PR TITLE
refactor(steps)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

Add tests for step create and get. Also some refactoring.
 Will leave the queries alone for now. As i'd like to have a database connection going to better understand what is being retrieved etc.

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/crud/test_step.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    """Test to create a step in the database."""
+    # GIVEN a valid step
+
+    # WHEN creating a step in the database
+    create_step(adapter=mock_adapter, step=valid_step)
+
+    # THEN a step can be retrieved from the database
+    step: dict = mock_adapter.step_collection.find_one({"prep_id": valid_step.prep_id})
+    assert step
+    assert isinstance(step, dict)
```
Comment (diitaz93) [inline tests/crud/test_step.py:18]: Maybe an assertion about the content of the dict would be nice

**Review by diitaz93 — APPROVED** [review]
Looks good 🎅

File: tests/crud/test_step.py (hunk 1 above)
Comment (ChrOertlin) [reply tests/crud/test_step.py:18]: fixed

**Comment by ChrOertlin** [comment]
tested by adding steps via lims-stage

**Pull request merged**